### PR TITLE
chore: Adds a schedule trigger to update-upstreams and updates Actions to use the trestlebot container image

### PR DESCRIPTION
**Pull request by jpower432**
## What Changed?

- Adds a schedule trigger for automated updates of third party content

Closes PSCE-304

Stacked with #68 

## Tested
- https://github.com/jpower432/oscal-profiles/actions/runs/7426722063
- https://github.com/jpower432/oscal-profiles/actions
- https://github.com/jpower432/oscal-profiles/pull/16

## Review Hints
- In order to test this I needed to update the FedRAMP REF in the workflow to the comment before the latest. The content was updated to OSCAL version `1.1.1` which is not supported by compliance-trestle yet. The is a temporary workaround until the content version is supported.

### CODE REVIEW

File: .github/workflows/update-upstream.yml [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+  # Schedule every Sunday at 5:00
+  schedule:
+    - cron: "0 5 * * 0"
 
 env:
   FEDRAMP_PROFILE_NAME: "fedramp_rev5_high"
   NIST_CATALOG_NAME: "nist_rev5_800_53"
-  FEDRAMP_REPO_REF: "master"
+  FEDRAMP_REPO_REF: "d9cbac30174fa9d52540eba34425bdb5c18ff4d8"
   NIST_REPO_REF: "690f517daaf3a6cbb4056d3cde6eae2756765620"
```
Comment (beatrizmcouto) [inline .github/workflows/update-upstream.yml:13]: Let's decide what to do after the community meeting

**Review by beatrizmcouto — COMMENTED** [review]
Let's wait for tomorrow until after the community meeting to decide what to do

**Comment by jpower432** [comment]
@beatrizmcouto Leaving this PR as is for now, but moving the container image changes to a new PR so we can keep the on-hold changes to a minimum.

**Comment by jpower432** [comment]
Closing in favor of #69

**Pull request closed without merging**